src/Modules/ModParser.lua

```diff
@@ -3633,7 +3636,7 @@ local flagTypes = {
 	["no life regeneration"] = "NoLifeRegen",
 	["hexproof"] = { name = "CurseEffectOnSelf", value = -100, type = "MORE" },
 	["hindered, with (%d+)%% reduced movement speed"] = "Condition:Hindered",
-	["unnerved"] = "Condition:Unnerved",
```

Comment:
this is here so that if you add more after this line in the future you dont need to mess with this "unrelated" line